aten/src/ATen/native/quantized/cpu/qlinear.cpp

```diff
@@ -165,16 +142,22 @@ class QLinearInt8 final : public torch::OperatorKernel {
       //  1) Add in row and column offsets to the rows and columns,
       //  respectively.
       //  2) Add in the bias term.
-      fbgemm::ReQuantizeOutput<ReluFused> outputProcObj(
-          /*nextop=*/doNothingObj,
-          /*C_multiplier=*/output_multiplier_float.data(),
-          /*C_zero_point=*/output_zero_point_int32,
-          /*Aq_zero_point=*/input_zero_point_int32,
-          /*Bq_zero_point=*/pack_ptr.w_zp.data(),
-          /*row_offsets=*/packA.getRowOffsetBuffer(),
-          /*col_offsets=*/col_offsets.data(),
-          /*bias=*/bias_ptr,
-          /*nCol=*/N);
+      fbgemm::ReQuantizeOutput<
+          ReluFused,
+          fbgemm::QuantizationGranularity::TENSOR,
+          float>
+          outputProcObj(
+              doNothingObj,
```

Comment:
Maybe keep the comments for better annotating these arguments.

Reply:
I removed the trivial ones and kept ones which provide some context.